practice/flutter/flutter/models.py

```diff
@@ -0,0 +1,14 @@
+"""flutter Models."""
+
+from django.db import models
+
+
+class Flutt(models.Model):
+    """A flutt class model that stores text and a timestamp."""
+    text = models.TextField()
+    timestamp = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.text
+    def __repr__(self):
```

Comment:
Doctests. You can manually specify `timestamp` to a constructor call so all of this will be deterministic.